### ISSUE TITLE
Script cannot resolve winget path when multiple instances of it exist

### ISSUE THREAD

**Issue by Matt-13** [Romanitho/Winget-AutoUpdate#39]
Hi Romain, I have a couple of machines that have been upgraded from Windows 10 to Windows 11 - used for about 4 years, same install. Script was working perfectly on Version 1.6.3, after the 1.7.0 update in updates.log the output is: "Winget isn't installed"
I work with multiple machines in a large company and stumbled upon your script which works very well for updating apps.

In Get-Wingetoutdatedapps.ps1:

`    #Get WinGet Path
    $WingetPath = (Resolve-Path "C:\Program Files\WindowsApps\Microsoft.DesktopAppInstaller_*_x64__8wekyb3d8bbwe").Path `

Result of $WingetPath is nothing/null.

There are multiple instances of Microsoft.DesktopAppInstaller under the WindowsApps folder.

```
PS C:\Program Files\WindowsApps> dir | Select-String Microsoft.DesktopAppInstaller

C:\Program Files\WindowsApps\Microsoft.DesktopAppInstaller_1.0.30311.0_x64__8wekyb3d8bbwe
C:\Program Files\WindowsApps\Microsoft.DesktopAppInstaller_1.17.10271.0_neutral_split.scale-100_8wekyb3d8bbwe
C:\Program Files\WindowsApps\Microsoft.DesktopAppInstaller_1.17.10271.0_neutral_split.scale-125_8wekyb3d8bbwe
C:\Program Files\WindowsApps\Microsoft.DesktopAppInstaller_1.17.10271.0_x64__8wekyb3d8bbwe
C:\Program Files\WindowsApps\Microsoft.DesktopAppInstaller_2019.131.908.0_neutral_~_8wekyb3d8bbwe
C:\Program Files\WindowsApps\Microsoft.DesktopAppInstaller_2022.127.2322.0_neutral_~_8wekyb3d8bbwe
```

Maybe needs an extra conditional statement that checks for multiple paths?

Any help is appreciated.
Cheers

**Comment by Romanitho** [comment]
Hi, thanks for your comment.
I didn't even know it was possible to have different versions, but anyway. I'll work on that and publish a fix as pre-release as soon as I can.

Thanks for details ;)

**Comment by Romanitho** [comment]
Can you test with pre-release 1.7.1 please ?

**Comment by Matt-13** [comment]
Works on 1.7.1

```
##################################################
#     CHECK FOR APP UPDATES - 7/04/2022
##################################################
05:00:17 - Notification Langugage : English
05:00:17 - Checking internet connection...
05:00:21 - Connected !
05:00:21 - WAU AutoUpdate is Enabled. Current version : 1.7.0
05:00:23 - WAU is up to date.
05:00:23 - WAU uses Black List config
05:00:23 - Checking application updates on Winget Repository...
05:00:23 - Winget not installed !

##################################################
#     CHECK FOR APP UPDATES - 8/04/2022
##################################################
05:00:02 - Notification Langugage : English
05:00:02 - Checking internet connection...
05:00:03 - Connected !
05:00:03 - WAU AutoUpdate is Enabled. Current version : 1.7.0
05:00:03 - WAU Available version: 1.7.1
05:00:08 - Downloading the GitHub Repository version 1.7.1
05:00:09 - Unzipping the WAU GitHub Repository
05:00:09 - Updating WAU
05:00:09 - Done. Cleaning temp files
05:00:09 - WAU Update completed.
05:00:13 - Re-run WAU

##################################################
#     CHECK FOR APP UPDATES - 8/04/2022
##################################################
05:00:14 - Notification Langugage : English
05:00:14 - Checking internet connection...
05:00:14 - Connected !
05:00:14 - WAU AutoUpdate is Enabled. Current version : 1.7.1
05:00:14 - WAU is up to date.
05:00:14 - WAU uses Black List config
05:00:14 - Checking application updates on Winget Repository...
Available update : Microsoft Visual C++ 2008 Redistributable - x86. Current version : 9.0.30729.4148. Available version : 9.0.30729.6161.
Available update : Logitech G HUB. Current version : 2021.3.5164. Available version : 2022.3.242300.
Available update : Microsoft Teams. Current version : 1.2.0.22654. Available version : 1.5.00.8070.
Available update : Microsoft Visual C++ 2008 Redistributable - x86. Current version : 9.0.30729. Available version : 9.0.30729.6161.
Available update : Microsoft Visual C++ 2008 Redistributable - x86. Current version : 9.0.30729. Available version : 9.0.30729.6161.
05:00:27 - Updating Microsoft Visual C++ 2008 Redistributable - x86 from 9.0.30729.4148 to 9.0.30729.6161...
05:00:32 - ##########   WINGET UPGRADE PROCESS STARTS FOR APPLICATION ID 'Microsoft.VC++2008Redist-x86'   ##########
```

**Comment by Romanitho** [comment]
Perfect. Thank you for confirmation. :)
